tests/tier2/tc_2047_check_rhsm_log_after_delete_host_hypervisor_from_server_webui.py

```diff
@@ -32,13 +32,13 @@ def test_run(self):
         try:
             vw_host_name = self.get_hostname(self.ssh_host())
             vw_host_uuid='xxx'
-            res1 = self.vw_web_host_delete(vw_host_name, vw_host_uuid)
+            self.vw_web_host_delete(vw_host_name, vw_host_uuid)
```

Comment:
It would be better to add the return for the `vw_web_host_delete()` function, what do you think of this?

Reply:
What should we add? About whether this function run successfully or not?

Reply:
Okay, after reviewing the following functions in the [virt_who/testing.py ](https://github.com/VirtwhoQE/virtwho-ci/blob/master/virt_who/testing.py#L1242)file, I think your way is better :)